src/azure-account.ts

```diff
@@ -3,213 +3,40 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-assignment
-const CacheDriver = require('adal-node/lib/cache-driver');
-// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-const createLogContext = require('adal-node/lib/log').createLogContext;
-
-import { SubscriptionClient, SubscriptionModels } from '@azure/arm-subscriptions';
```

Comment:
Can we get a quick "outline" of what you want the resulting file structure to look like after you've moved all files and after you've added msal code? None of these names seem bad, but it's hard to get a big picture idea of what the end goal is.

I don't really care how you create the outline, but personally I would probably mess around with some empty files/folders in VS Code to get a screenshot like this:
![Screen Shot 2021-06-11 at 10 48 09 AM](https://user-images.githubusercontent.com/11282622/121728862-a95eb100-caa2-11eb-843b-0b73f779634a.png)

Reply:
Here's a rough look at what I'm thinking: 

![Screen Shot 2021-06-11 at 11 28 20 AM](https://user-images.githubusercontent.com/22795803/121733891-ea59c400-caa8-11eb-8f37-cc1429cb9393.png)